java/pom.xml

```diff
@@ -199,6 +200,15 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
why do you need to add this?

Reply:
Oh my bad...I couldnt compile due to the java version. I checked it out to master